frontend/src/js/components/modals/Calculator/InputArea.jsx

```diff
@@ -63,7 +63,12 @@ class InputArea extends React.Component {
                 <div id="assignment">
                     <span className="glyphicon glyphicon-arrow-left" />
                 </div>
-                <input id="calculation" type="text" value={this.props.calculation} disabled />
+                <input 
```

Comment:
I like this style a lot more, much more readable then having everything in one line.